### ISSUE TITLE
Alan Wake 2 can't turn on FG in game settings menu.

### ISSUE THREAD

**Issue by Dorennor** [Nukem9/dlssg-to-fsr3#134]

**Comment by Dorennor** [comment]
![зображення](https://github.com/Nukem9/dlssg-to-fsr3/assets/40647609/55910a08-a095-43c7-a52d-d0e7ee4a51a1)
![Alan Wake 2 Screenshot 2023 12 20 - 00 57 57 20-sdr](https://github.com/Nukem9/dlssg-to-fsr3/assets/40647609/56f71010-bf00-426b-990b-3af35242897f)

**Comment by Dorennor** [comment]
Lenovo Legion 5 2021 
RTX 3070 Laptop 130w
Ryzen 7 5800H

**Comment by WadeWayne1352** [comment]
I can't activate it either

**Comment by Igihara** [comment]
Use dlsstweaks version

**Comment by Vladzor87do** [comment]
![Screenshot 2023-12-20 004253](https://github.com/Nukem9/dlssg-to-fsr3/assets/63660560/b632607d-fcf8-46fe-b39f-44604c822dff)
![Screenshot 2023-12-20 004225](https://github.com/Nukem9/dlssg-to-fsr3/assets/63660560/86499faf-9a49-44c4-b6e2-7f1ab76c1f93)

Its working, non-tweaks version

**Comment by Asso83** [comment]
> ![Screenshot 2023-12-20 004253](https://private-user-images.githubusercontent.com/63660560/291740157-b632607d-fcf8-46fe-b39f-44604c822dff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMwMzA2MjMsIm5iZiI6MTcwMzAzMDMyMywicGF0aCI6Ii82MzY2MDU2MC8yOTE3NDAxNTctYjYzMjYwN2QtZmNmOC00NmZlLWIzOWYtNDQ2MDRjODIyZGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE5VDIzNTg0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5NTFjYzUwMzEzODA4M2E0NTYyNjBhZGFiOWExNmNhYjBmNDJiYzYyZmE3ZmMyZjUyMjRiNzk2YjQ5MDgwY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lmSbd9rRn8aToGJKE1Es3Wd08ZQ8c2E4vD5OZa9lTuM) ![Screenshot 2023-12-20 004225](https://private-user-images.githubusercontent.com/63660560/291740170-86499faf-9a49-44c4-b6e2-7f1ab76c1f93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMwMzA2MjMsIm5iZiI6MTcwMzAzMDMyMywicGF0aCI6Ii82MzY2MDU2MC8yOTE3NDAxNzAtODY0OTlmYWYtOWE0OS00NGM0LWI2ZTItN2YxYWI3NmMxZjkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE5VDIzNTg0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4YjdmOGU2ZmJkNDlkOWY0YjY3ZTE1N2UwODQxOTk3MDI5ZGIwYWVmMDcwNmY1YmM3ODI1N2FiNzg3MGU2OGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lgJpXkGYe8Dik2g8jUUUjd8hdVUDQtmmiTXa9K0HJ6o)
> 
> v0.60 Its working, non-tweaks version RTX3080

You have AW2 EPIC GAMES original game ? The mod don't work in my 3080.  Cyberpunk e other games from steam and GOG work fine, but at least on my card AW2 Mod don't work.

**Comment by Vladzor87do** [comment]
> You have AW2 EPIC GAMES original game ? The mod don't work in my 3080. Cyberpunk e other games from steam and GOG work fine, but at least on my card AW2 Mod don't work.

yes its EPIC version on RTX3080, just drop the two files in to the game folder (non tweaks version)

**Comment by Asso83** [comment]
For all people use EPIC GAMES, delete all file in this folder
C:\Program Files (x86)\Epic Games\Launcher\Portal\Extras\Overlay
... and FG work.

**Comment by ArthurFiNCH** [comment]
> For all people use EPIC GAMES, delete all file in this folder C:\Program Files (x86)\Epic Games\Launcher\Portal\Extras\Overlay ... and FG work.

Have you actually tried it? Because I did. It does not work.

**Comment by Squirrel-767** [comment]
> For all people use EPIC GAMES, delete all file in this folder C:\Program Files (x86)\Epic Games\Launcher\Portal\Extras\Overlay ... and FG work.

Thank you so much! It worked with this.

**Comment by Nooblet16** [comment]
Yea this has been an issue since the first release.
The Tweaks method makes the option available but the dll method keeps it greyed out so you have to turn it on from ini file.

It's odd that Epic Overlay has anything to do with it.
